filesystem.mod/filesystem.bmx

```diff
@@ -284,6 +284,73 @@ Function SetFileTime( path:String, time:Long, timeType:Int=FILETIME_MODIFIED)
 	End If
 End Function
 
+Rem
+bbdoc: Sets the file modified or last accessed time.
+about: @dateTime is the basic DateTime struct defined in pub.stdc .
+End Rem
+Function SetFileTime( path:String, dateTime:SDateTime, timeType:Int=FILETIME_MODIFIED)

```

Comment:
There's no need to duplicate all the code each time. You can just call something like `SetFileTime( path, dateTime.ToEposhSecs(), timeType)`

Reply:
Incorporated your suggestion.